python/semantic_kernel/connectors/memory/qdrant/qdrant_memory_store.py

```diff
@@ -0,0 +1,385 @@
+            key {str} -- The unique database key of the record.
+            with_embedding {bool} -- Whether to include the embedding in the result. (default: {False})
+
+        Returns:
+            MemoryRecord -- The record.
+        """
+        
+        collection_info = self._qdrantclient.get_collection(collection_name=collection_name)
+        with_payload = True
+        search_id = [key]
+
+        if not collection_info.status == CollectionStatus.GREEN:
+            raise Exception(f"Collection '{collection_name}' does not exist")
+
+        qdrant_record = self._qdrantclient.retrieve(
+            collection_name=collection_name,
+            ids=search_id,
+            with_payload=with_payload,
+            with_vectors=with_embedding
+        )
+
+        result = MemoryRecord(
+            is_reference=False, 
+            external_source_name="qdrant", 
+            key=search_id, 
+            id=search_id, 
+            embedding=qdrant_record.vector, payload=qdrant_record.payload)
+        
+        return result
+        
+
+    async def get_batch_async(
+        self, collection_name: str, keys: List[str], with_embeddings: bool = False
+    ) -> List[MemoryRecord]:
+        """Gets a batch of records.
+
+        Arguments:
+            collection_name {str} -- The name of the collection to get the records from.
+            keys {List[str]} -- The unique database keys of the records.
+            with_embeddings {bool} -- Whether to include the embeddings in the results. (default: {False})
+
+        Returns:
+            List[MemoryRecord] -- The records.
+        """
+        
+        collection_info = self._qdrantclient.get_collection(collection_name=collection_name)
+
+        if not collection_info.status == CollectionStatus.GREEN:
+            raise Exception(f"Collection '{collection_name}' does not exist")
+        
+        with_payload = True
+        search_ids = [keys]
+
+
+        qdrant_records = self._qdrantclient.retrieve(
+            collection_name=collection_name,
+            ids=search_ids,
+            with_payload=with_payload,
+            with_vectors=with_embeddings
+        )
+
+        return qdrant_records
+
+    async def remove_async(self, collection_name: str, key: str) -> None:
+        """Removes a record.
+
+        Arguments:
+            collection_name {str} -- The name of the collection to remove the record from.
+            key {str} -- The unique database key of the record to remove.
+
+        Returns:
+            None
+        """
+        
+        collection_info = self._qdrantclient.get_collection(collection_name=collection_name)
+        
+        if not collection_info.status == CollectionStatus.GREEN:
+            raise Exception(f"Collection '{collection_name}' does not exist")
+
+        self._qdrantclient.delete(
+            collection_name="{collection_name}",
+            points_selector=models.PointIdsList(
+            points=[key],
```

Comment:
see comment about key vs point ID

Reply:
I added function to for memory record conversion, if this works for key vs. pointid I can change these to leverage that pointid

Reply:
Resolved in https://github.com/microsoft/semantic-kernel/pull/919/commits/67f0898cddb7d290bb4b6511f7aaeeceaa63cd7a